docs/reference/tools-reference/new-llm-tool.md

```diff
@@ -0,0 +1,84 @@
+# LLM 
+
+## Introduction
+Prompt flow LLM tool enables you to leverage widely used large language models like [OpenAI](https://platform.openai.com/) or [Azure OpenAI (AOAI)](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/overview) for natural language processing. 
+
+Prompt flow provides a few different LLM APIs:
+- **[Completion](https://platform.openai.com/docs/api-reference/completions)**: OpenAI's completion models generate text based on provided prompts.
+- **[Chat](https://platform.openai.com/docs/api-reference/chat)**: OpenAI's chat models facilitate interactive conversations with text-based inputs and responses.
+
+
+## Prerequisite
+Create OpenAI or Azure OpenAI resources:
+
+- **OpenAI**
+
+    Sign up account [OpenAI website](https://openai.com/)
+
+    Login and [Find personal API key](https://platform.openai.com/account/api-keys)
+
+- **Azure OpenAI (AOAI)**
+
+    Create Azure OpenAI resources with [instruction](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource?pivots=web-portal)
+
+## **Connections**
+
+Setup connections to provisioned resources in prompt flow.
+
+| Type        | Name     | API KEY  | API Type | API Version |
+|-------------|----------|----------|----------|-------------|
+| OpenAI      | Required | Required | -        | -           |
+| AzureOpenAI | Required | Required | Required | Required    |
+
+
+## Inputs
```

Comment:
Why no `connection` and `api` in inputs?

Reply:
Since `connection` and `api` are sections that are written outside of the `inputs` section. Just follow the existing structure